### DISCUSSION TITLE
Nomad title/rank

### DISCUSSION THREAD

**Discussion by dragontekgaming** [TownyAdvanced/Towny#5770] [Q&A] [answered]
Is it possible to make it so Nomads show a title/rank in chat?

**Comment by LlmDl** [comment, marked as answer]
This placeholder might be what you're after:

%townyadvanced_player_status% - Displays Nomad, Resident, Mayor or King, depending on what position the player is in.

**Comment by dragontekgaming** [comment]
Thank you, this was it.
